app/app/src/main/java/org/omecproject/up4/config/Up4Config.java

```diff
@@ -38,18 +38,28 @@
     public static final String UE_POOLS = "uePools";
     public static final String S1U_ADDR = "s1uAddr";
     public static final String N3_ADDR = "n3Addr";
-
+    // Must be provided if using uePool, s1uAddr or n3Addr to configure N3 and N6 interfaces
+    public static final String SLICE_ID = "sliceId";
 
     @Override
     public boolean isValid() {
         return hasOnlyFields(DEVICE_ID, DEVICES, UE_POOLS, S1U_ADDR, N3_ADDR,
-                             DBUF_DRAIN_ADDR, MAX_UES, PSC_ENCAP_ENABLED) &&
+                             DBUF_DRAIN_ADDR, MAX_UES, PSC_ENCAP_ENABLED,
+                             SLICE_ID) &&
                 // Mandatory fields.
                 (hasField(DEVICE_ID) || hasField(DEVICES)) &&
+                isSliceIdFieldRequired() &&
                 !upfDeviceIds().isEmpty() &&
                 isDbufConfigValid();
     }
 
+    private boolean isSliceIdFieldRequired() {
+        if (hasField(UE_POOLS) || hasField(S1U_ADDR) || hasField(N3_ADDR)) {
+            return hasField(SLICE_ID);
+        }
+        return true;
```

Comment:
false ?

Reply:
no, slice ID is required only if one of the other configurations is in the netcfg, otherwise, no need to specify slice ID.